src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -696,22 +696,32 @@ export class BaseEditCourseForm extends React.Component {
             courseSubmitting={submitting}
             {...this.props}
           />
+          {(!courseInReview && pristine &&
+          <Link to={`/courses/${uuid}/rerun`}>
+            <button
+              className="btn btn-block rounded mt-3 new-run-button"
+            >
+              <Icon className="fa fa-plus" /> Add Course Run
+            </button>
+          </Link>
+          )}
+          {(!pristine &&
```

Comment:
Should this also have the `!courseInReview` check? Otherwise we will show the disabled button when the course is in review if the form is not pristine. Or is it safe because we disable editing the form once it is in review so it should remain pristine?

Might have weird original behaviour when you first submit for review because that will change `courseInReview` to true so the working link won't be displayed, but since we don't make the form pristine again, the disabled button will still show and then when they refresh the page, it will clear back to not showing anything.